staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere.go

```diff
@@ -894,7 +901,16 @@ func (vs *VSphere) LoadBalancer() (cloudprovider.LoadBalancer, bool) {
 }
 
 func (vs *VSphere) isZoneEnabled() bool {
-	return vs.cfg != nil && vs.cfg.Labels.Zone != "" && vs.cfg.Labels.Region != ""
+	isEnabled := vs.cfg != nil && vs.cfg.Labels.Zone != "" && vs.cfg.Labels.Region != ""
```

Comment:
Is it really both zone and region need to be set or just one of zone and region needs to be set?

Reply:
https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/zones.html this doc tells that both need to be set. Case with one parameter does not described at all.